source/_integrations/switchbot_cloud.markdown

```diff
@@ -41,6 +44,7 @@ Please note, device names configured in the SwitchBot app are transferred into H
 - Meter
 - MeterPlus
 - Outdoor Meter
+- Vacuum K10+, K10+ pro, S1, S1 Plus
 
```

Comment:
Please add a section ## Entities ### Vacuum, and list all the supported actions, like Neato does, for example.